src/lib/utils/options/sources/typedoc.ts

```diff
@@ -26,8 +26,9 @@ export function addTypeDocOptions(options: Options) {
         help: "Specifies the output mode the project is used to be compiled with: 'file' or 'modules'",
```

Comment:
Update help text?

Reply:
Should also probably update readme file as well:

> `--mode <file|modules>`

Reply:
Good catch, thanks!

Reply:
Good catch, thanks!

Reply:
Good catch, thanks!

Reply:
Website docs will also be updated by https://github.com/TypeStrong/typedoc-site/pull/26